File: BE-FindingJob/src/main/java/com/example/befindingjob/service/SkillService.java
package com.example.befindingjob.service;

import com.example.befindingjob.entity.Skill;
import com.example.befindingjob.model.ApiResponse;

import java.util.Set;

public interface SkillService {
    ApiResponse<Void> updateSkills(Integer jobId, Set<Skill> skills);
}

File: JobHub/app/src/main/java/com/example/jobhub/service/SkillService.kt
package com.example.jobhub.service

import com.example.jobhub.entity.Skill
import com.example.jobhub.model.ApiResponse
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.POST
import retrofit2.http.Query

interface SkillService {
    @POST("api/skill/update")
    fun updateSkills(
        @Query("jobId") jobId: Int,
        @Body skills: Set<Skill>
    ): Call<ApiResponse<Void>>
}